pymedphys/labs/quickcheck/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright [2020] [Rafael Ayala]
+
```

Comment:
Would you be able to delete this license file and instead include this as a header in all your source files?

See the following for how other files have it done:

https://github.com/pymedphys/pymedphys/blob/32fc4275251659ceef68bdb49aab7da6245696d5/pymedphys/cli/main.py#L1-L13